caffe2/core/context_gpu.cu

```diff
@@ -428,7 +425,15 @@ struct DefaultCUDAAllocator final : public at::Allocator {
         }
         return {ptr, ptr, &Delete, at::Device(CUDA, CaffeCudaGetDevice())};
       case CudaMemoryPoolType::THC:
-        CUDA_ENFORCE(g_thc_allocator->Alloc(&ptr, nbytes, 0 /* stream */));
+        {
+          // TODO: eliminate this
```

Comment:
I'd leave a comment here still, describing that it's a broken behavior and we just preserving it and want to eliminate it in the follow up diff. Otherwise I'm worried that we'd need to retrace our steps again in a few weeks.